Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'jbuilder', '~> 2.5'
 gem 'omniauth'
 gem 'omniauth-fitbit'
 gem "figaro"
+gem 'fitbit-api-client', require: 'fitbit'
```

Comment:
Hey Bryce, there doesn't seem to be well maintained Fitbit client that is significantly better than the one I am using (I can't see one on RubyGems anyway). Potentially we could fork this gem and maintain it ourselves or we could write our own. How do you want to proceed?